php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
```

Comment:
Typo inside comment in token 'witout'. Possible corrections:without (85),inout (0),witout (0)